### PR TITLE
Pubsub query

### PR DESCRIPTION
**Pull request by wyfo**
See https://github.com/eclipse-zenoh/zenoh/pull/1193

### CODE REVIEW

File: src/qos.rs
Comment (Mallets) [inline src/qos.rs:1]: Add license header.

File: src/query.rs
Comment (Mallets) [inline src/query.rs:1]: Add license header.

File: tests/stubs_check.py
Comment (Mallets) [inline tests/stubs_check.py:1]: Add license header.

**Review by Mallets — CHANGES_REQUESTED** [review]
LGTM. Just add license headers.

**Comment by wyfo** [comment]
This PR is based on the previous ones, I've fixed licence headers in #243

**Comment by wyfo** [comment]
I will need to rebase when #243 will be merged

**Review by Mallets — APPROVED** [review]

**Pull request merged**